### DISCUSSION TITLE
IOS 16功能更新!📱

### DISCUSSION THREAD

**Discussion by bonny123865** [1101-GS4719-C/Python-Community-GS4719-C#688] [General]
新聞來源:
https://www.elle.com/tw/life/tech/g41241002/ios16/

原文部分內容:
你更新 iOS 16 了嗎？這次的更新中最具特色的非「鎖定畫面」莫屬了，可以任意更換字體大小和字型、隨機顯示手機中的照片，甚至改變了通知的顯示方式！除了這項有趣的變更外，還有我們期待已久的內建功能「照片去背」，另外「訊息」、「郵件」、「原況文字」和「圖像查詢」也都有相當新奇的改變，新增的「iCloud 共享圖庫」更是好友間分享照片的超棒設計！
不論你是否已經更新，快來看看這次 iOS 16 總共推出了哪些新奇又實用的功能吧！

個人看法:
裡面有介紹很多超實用的隱藏功能!
我個人認為一鍵去背真的很實用，而且可以直接剪下放到LINE的聊天室內，完全不用PS去背，無論是人物或是物體都可以喔
手拿iphone的各位可以更新看看~

![image](https://user-images.githubusercontent.com/90883526/191437238-b16ca32b-81ef-4951-98d8-001730c4c138.png)

**Comment by Valentinaa0214** [comment]
更新後變好醜QQ而且也好耗電...

**Reply by ellieleung12** [reply]
變耗電有同感QQ

**Reply by bonny123865** [reply]
但是我個人認為還不錯，可以直接在鎖定畫面上看到電量跟天氣!

**Reply by Valentinaa0214** [reply]
@bonny123865 買14給我🥲🥲

**Comment by Yunhans** [comment]
照片可以直接去背是不是有用到ai的技術

**Comment by Chiway1229** [comment]
我覺得IOS16的主畫面超醜的...
那個時鐘字體肥肥的 掉電速度也變好快

**Reply by ellieleung12** [reply]
那個字體可以改喔~長按鎖定螢幕可以調其他字體！

**Comment by a45320614** [comment]
我是用安卓的手機，超級想要這個功能QQ
不知道有沒有人可以贊助我

**Comment by williamyen0623** [comment]
IOS16真香 主畫面變超美

**Comment by AnAnistired** [comment]
怎麼這麼多人覺得新桌面變醜，我覺得很好看欸，但自訂性的部分似乎還是比安卓少了許多，小工具好少🥲

**Comment by ABCDEFGHIJKLMNO12345678** [comment]
除了掉電速度變得比之前還快外 主畫面的字體也做的不盡理想

**Comment by Pietra1226** [comment]
最近也更新了，鎖定畫面真的蠻美

**Comment by danwan1027** [comment]
i16 更新真滴香

**Comment by ABCDEFGHIJKLMNO12345678** [comment]
看來我現在用的SE還沒有被時代逃汰

**Comment by zhiron544** [comment]
我覺得一鍵去背和 iCloud 共享圖庫是很棒的設計，可以方便我們處理和分享照片。我也想知道鎖定畫面的字體和照片是否可以自由選擇，還是有固定的模式。
